[2024-07-13T18:48:33.401+0530] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: raw_load_loans.create_table manual__2024-07-13T13:18:31.231660+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
[2024-07-13T18:50:32.981+0530] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: raw_load_loans.create_table manual__2024-07-13T13:18:31.231660+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
[2024-07-13T18:50:47.243+0530] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: raw_load_loans.create_table manual__2024-07-13T13:18:31.231660+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
[2024-07-13T18:58:35.591+0530] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: raw_load_loans.create_table manual__2024-07-13T13:18:31.231660+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
